sparkler-api/src/main/java/edu/usc/irds/sparkler/Constants.java

```diff
@@ -59,28 +62,49 @@
         // Parser Properties
 
         // Plugin Properties
+        @ConfigKey
+        String PLUGINS = "plugins";
     }
 
 
     abstract class defaults {
         /**
          * Create configuration instance for Sparkler
          */
-        public static Configuration newDefaultConfig(){
-
-            Configuration conf = new Configuration();
-            conf.set(key.UUID_KEY, UUID.randomUUID().toString());
-            conf.addResource(file.SPARKLER_DEFAULT);
-            conf.addResource(file.SPARKLER_SITE);
-            return conf;
+        public static SparklerConfiguration newDefaultConfig(){
+            Yaml yaml = new Yaml();
+            InputStream input = null;
+            SparklerConfiguration sparklerConf = null;
+            try {
+                input = Constants.class.getClassLoader().getResourceAsStream(file.SPARKLER_DEFAULT);
+                Map<String,Object> yamlMap = (Map<String, Object>) yaml.load(input);
+                sparklerConf = new SparklerConfiguration(yamlMap);
+
+                //input = Constants.class.getClassLoader().getResourceAsStream(file.SPARKLER_SITE);
+                //if(sparklerSite != null)
+                //    sparklerConf.mask(sparklerSite);
+            } catch (Exception e) {
+                e.printStackTrace();
+            } finally {
+                if(input != null)
+                    try {
+                        input.close();
```

Comment:
Use `IOUtils.closeQuietly(input)` and remove try-catch-nullcheck in finally block
https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/IOUtils.html#closeQuietly(java.io.Closeable)